rules/integrations/o365/defense_evasion_microsoft_365_mailboxauditbypassassociation.toml

```
@@ -0,0 +1,59 @@
+[metadata]
+creation_date = "2022/01/13"
+integration = "o365"
+maturity = "production"
+updated_date = "2022/01/13"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects the occurrence of mailbox audit bypass associations. The mailbox audit is responsible for logging specified
+mailbox events (like accessing a folder or a message or permanently deleting a message). However, actions taken by some
+authorized accounts, such as accounts used by third-party tools or accounts used for lawful monitoring, can create a
+large number of mailbox audit log entries and may not be of interest to your organization. Because of this,
+administrators can create bypass associations, allowing certain accounts to perform their tasks without being logged.
+Attackers can abuse this allowlist mechanism to conceal actions taken, as the mailbox audit will log no activity done by
+the account.
+"""
+false_positives = ["Legitimate whitelisting of noisy accounts"]
+from = "now-30m"
+index = ["filebeat-*", "logs-o365*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Mailbox Audit Logging Bypass"
```

```suggestion
name = "O365 Mailbox Audit Logging Bypass"
```

Or maybe spell it out